blog/_src/posts/2024-05-17-racket-v8-13.md

```diff
@@ -1,28 +1,31 @@
     Title: Racket v8.13
-    Date: 2024-05-17T11:41:54
-    Tags: DRAFT
+    Date: 2024-05-16T23:06:42
+    Tags: release-announcement, DRAFT
+
+*posted by Stephen De Gabrielle*
 
 ----------------------------------------------------------------------
 
+# Racket v8.13 is now available
```

Comment:
I would recommend against using heading level 1 here, since the post title is already an (implicit) heading level 1.

Reply:
done

Reply:
Thank you . 
I have fixed here and in discourse post. Look ok to you?

Reply:
LGTM